doc/debian/jitsi-meet/jitsi-meet.example

```diff
@@ -39,6 +39,15 @@ server {
     index index.html index.htm;
     error_page 404 /static/404.html;
 
+    gzip on;
```

Comment:
perhaps add `gzip_vary on;` too?

Reply:
Sure, that seems a useful enough addition.